CDCgov/prime-reportstream
# Intrivo - ORC-22-4 Blank

This PR addresses the issue where OTC data came without ORC-22-4 (ordering_facility_state) and this was causing our api filters to exclude these OTC messages, coming from intrivo. The solution was to use the zip code lookup mapper to display the state.

Test Steps:
1. Create fake data for intrivo
2. Blank out ORC-22-4
3. Fire up docker
4. reloadtables and reloadsettings
5. Send to api and verify generated files have the state in ORC-22-4
6. Repeat with fake data having ORC-22-4 populated

## Changes
- added new entry to hl7-ingest-covid-19.schema: `-name: ordering_facility_zip_lookup` and `-name: ordering_facility_state`

## Checklist

### Testing

## Linked Issues
- Fixes #6780
